compiler/compile/VirtualGuard.cpp

```diff
@@ -213,6 +213,60 @@ TR_VirtualGuard::createMethodGuard
    return createMethodGuardWithReceiver (kind, comp, calleeIndex, callNode, destination, calleeSymbol, thisClass, callNode->getSecondChild());
    }
 
+/*
+ * Shape of a breakpoint guard:
+ * iflcmpeq
+ *    land
+ *       aloadi <ConstantPool shadow>
+ *          aconst J9Method
+ *       isBreakpointedBit 
+ *    isBreakpointedBit 
+ */
+TR::Node*
+TR_VirtualGuard::createBreakpointGuard
+(TR::Compilation * comp, int16_t calleeIndex,
+ TR::Node* callNode, TR::TreeTop * destination, TR::ResolvedMethodSymbol * calleeSymbol)
+   {
+#ifdef J9_PROJECT_SPECIFIC
+   TR::SymbolReferenceTable * symRefTab = comp->getSymRefTab();
+   TR::SymbolReference * addressSymRef = symRefTab->findOrCreateJ9MethodConstantPoolFieldSymbolRef(offsetof(struct J9Method, constantPool));
+   TR::Node * aconstNode = TR::Node::aconst(callNode, (uintptrj_t)calleeSymbol->getResolvedMethod()->getPersistentIdentifier());
+   TR::Node * constantPool = TR::Node::createWithSymRef(TR::aloadi, 1, 1, aconstNode, addressSymRef);
+   aconstNode->setIsMethodPointerConstant(true);
+   aconstNode->setInlinedSiteIndex(calleeIndex);
+   aconstNode->setByteCodeIndex(0);
+
+   TR::Node *guard = NULL;
+   if (TR::Compiler->target.is64Bit())
+      {
+      TR::Node * flagBit = TR::Node::create(callNode, TR::lconst, 0, 0);
+      flagBit->setLongInt(comp->fej9()->offsetOfMethodIsBreakpointedBit());
+
+      guard =  TR::Node::createif(TR::iflcmpeq,
```

Comment:
Can you make the formatting in this if block match the else so that it is more obvious they are equivalent? An alternative to save code duplication might be just to create two opcode variables to hold the if and and and and then generate the trees which could prevent the 32 and 64 cases from diverging over time...